src/pages/signin/LoginForm.js

```diff
@@ -125,6 +134,12 @@ function LoginForm(props) {
             return;
         }
 
+        // If the user has entered a guide email, then we are going to enable an experimental Onyx mode to help with performance
+        if (PolicyUtils.isExpensifyGuideTeam(loginTrim)) {
+            Log.info('Detected guide email in login field, setting memory only keys.');
+            setEnableMemoryOnlyKeys();
```

Comment:
Do you know if it could cause a problem to call `window.enableMemoryOnlyKeys()` twice? I'm thinking about if the login fails and then the guide tries again.

Reply:
I do not think so from reading the code